README.md

```diff
@@ -1,17 +1,42 @@
-# NeuroID Mobile SDK for iOS
+# NeuroID iOS Mobile SDK
 
-NeuroID's Mobile SDK makes it simple to embed behavioral analytics inside your mobile app. With a few lines of code, you can connect your app with the NeuroID platform and make informed decisions about your users.
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Neuro-ID_neuroid-ios-sdk&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Neuro-ID_neuroid-ios-sdk)
 
-## Installation
+![Neuro-ID Logo](/docs//images/NeuroID_Logo.png)
 
```

Comment:
Any reason for not linking the cocoapods url here https://cocoapods.org/pods/NeuroID? I like how you did it for Jitpack.